builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -455,6 +455,68 @@ func TestAccVSphereVirtualMachine_createWithCdrom(t *testing.T) {
 	})
 }
 
+func TestAccVSphereVirtualMachine_createWithExistingVmdk(t *testing.T) {
```

Comment:
I do love it when I see all of the plus signs that add a unit test.  Cookie for u!!!